Cargo.toml

```diff
@@ -23,7 +23,7 @@ native-tls = ["hyper-tls"]
 rustls-tls = ["hyper-rustls"]
 
 [dependencies]
-webdriver = { version = "0.45", default-features = false }
+webdriver = { version = "0.46", default-features = false }
```

Comment:
Nice! I think this also means we can finally remove the `_time01` dependency a bit further down — would you mind doing that in this PR too?

Reply:
Good call! I think that you're right. I removed this and the lockfile cleaned itself up quite a bit! I've pushed a commit handling this.